examples/src/main/scala/org/apache/spark/examples/ml/ALSExample.scala

```diff
@@ -28,7 +28,7 @@ object ALSExample {
 
   // $example on$
   case class Rating(userId: Int, movieId: Int, rating: Float, timestamp: Long)
-  object Rating {
+  object RatingUtil {
```

Comment:
Is this object even needed? there's no reason this couldn't just be defined in main?